.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,7 @@
 		"dockerfile": "Dockerfile",
 		"args": {
 			// Update the VARIANT arg to pick a version of Go
-			"VARIANT": "1.19",
+			"VARIANT": "1.20",
```

Comment:
I don't use the `devcontainer` setup and so haven't tested this line here. Can somebody who uses this test it please?

Reply:
I was able to test the devcontainer and it worked fine with this version update.